init.lua

```diff
@@ -191,7 +191,8 @@ dofile(modpath.."/register.lua")
 if minetest.settings:get_bool("unified_inventory_bags") ~= false then
 	dofile(modpath.."/bags.lua")
 end
-
-dofile(modpath.."/item_names.lua")
+if minetest.settings:get_bool("unified_inventory_enable_item_names") == true then
```

Comment:
```suggestion
if minetest.settings:get_bool("unified_inventory_enable_item_names", true) then
```

or

```suggestion
if minetest.settings:get_bool("unified_inventory_enable_item_names") ~= false then
```

Otherwise it will default to disabled.